controlplane/src/core/repositories/analytics/AnalyticsRequestViewRepository.ts

```diff
@@ -112,8 +112,12 @@ export class AnalyticsRequestViewRepository {
       unit: Unit.Nanoseconds,
       title: 'P95 Latency',
     },
-    errorsWithRate: {
-      title: 'Errors (Rate%)',
+    errors: {
+      title: 'Errors',
+    },
+    errorRate: {
+      title: 'Error Rate%',
```

Comment:
`Error Rate %` or `Error Rate in %` but `Rate%` looks a like typo 😄

Reply:
Yeah typo, its supposed to be just `Error Rate`